controllers/operatorpolicy_controller_test.go

```diff
@@ -48,11 +48,11 @@ func TestBuildSubscription(t *testing.T) {
 
 	// Check values are correctly bootstrapped to the Subscription
 	ret, err := buildSubscription(testPolicy, nil)
-	assert.Equal(t, err, nil)
+	assert.NoError(t, err)
 	assert.Equal(t, ret.GroupVersionKind(), desiredGVK)
-	assert.Equal(t, ret.ObjectMeta.Name, "my-operator")
-	assert.Equal(t, ret.ObjectMeta.Namespace, "default")
-	assert.Equal(t, ret.Spec.InstallPlanApproval, operatorv1alpha1.ApprovalManual)
+	assert.Equal(t, "my-operator", ret.ObjectMeta.Name)
+	assert.Equal(t, "default", ret.ObjectMeta.Namespace)
+	assert.Equal(t, operatorv1alpha1.ApprovalManual, ret.Spec.InstallPlanApproval)
```

Comment:
This linter might be a pain as we enable it, but I think it's going to be very helpful to have all of these right if we have to debug a test